xbmc/video/VideoDatabase.cpp

```diff
@@ -5907,7 +5907,7 @@ void CVideoDatabase::EraseVideoSettings(const CFileItem &item)
   {
     std::string sql = PrepareSQL("DELETE FROM settings WHERE idFile=%i", idFile);
 
-    CLog::Log(LOGINFO, "Deleting settings information for files %s", item.GetPath().c_str());
+    CLog::Log(LOGINFO, "Deleting settings information for files %s", CURL::GetRedacted(CURL::GetRedacted(item.GetPath()).c_str()));
```

Comment:
Isn't calling `CURL::GetRedacted(...)` once enough?

Reply:
Right, and back then I also mentioned that to @notspiff but I forgot about and he didn't respond.